webpack/scenes/ContentViews/components/EnvironmentPaths/EnvironmentPaths.js

```diff
@@ -47,16 +49,16 @@ const EnvironmentPaths = ({
               {index === 0 && <hr />}
               <FormGroup key={`fg-${index}`} isInline fieldId="environment-checkbox-group">
                 {environments.map(env =>
-                  (<Checkbox
+                  (<CheckboxOrRadio
```

Comment:
I was pleasantly surprised that this is the only change needed to add radio buttons. Turns out `<Checkbox>` and `<Radio>` both take the same props that we're passing here.